persistence/rocksdb/src/main/java/org/infinispan/persistence/rocksdb/RocksDBStore.java

```diff
@@ -1077,4 +993,18 @@ private void putExpireDbData(ExpiryEntry entry) throws RocksDBException {
          expiredDb.put(expiryBytes, entry.keyBytes);
       }
    }
+
+   /*
+    * Instead of iterate in RocksIterator we use the first and last byte array
+    */
+   private void deleteRange(ColumnFamilyHandle handle) {
```

Comment:
I would suggest changing this method as it is really just clearing all entries for this column family handle.